Add minutes from 2025-10-16 AGM (#8)
sig-rpc/minutes: 2025/2025-10-16-agm.md

# 2025-10-16 AGM

2025 Annual General meeting

Agenda: [sig-rpc/minutes#7](https://github.com/sig-rpc/minutes/issues/7)

## Attendees

- (RC) Robert Chisholm [chair] (Sheffield)
- (PH) Peter Heywood [secretary] (Sheffield)

- (JM) Jost Migenda (KCL)
- (FS) Frederick Sonnenwald (Sheffield)
- (A) Alex (Oxford)
- (SC) Sam Cunliffe (UCL)
- (IB) Iain Barrass (Glasgow)
- (ST) Sam Tygier (STFC)
- (AD) Abshishek Dasgupta (Oxford)

- Welcome & introductions
    - > chance for any new attendees to be introduced to steering group
    - RC introduced the SIG, original committee etc. 
    - RC explained the origins
    - Attendees introduced themselves, why they are interested in the SIG
- Review previous minutes
    - Add minutes from `2025-07-08` meeting [sig-rpc/minutes#6](https://github.com/sig-rpc/pulls/issues/6)
    - JM to look it over and leave a comment/Review the GitHub after the meeting
- Review of the year's activities
    - SSI CW/N8 RSE Meetup (Robert)
        - RC delivered some ligtning talks at these events (short talks with a few slides) advertising the SIG
    - EuroSciPy (Jost)

>         - JM presented at EuroSciPy 2025, gaining some visibility in europe

Reviewer: ```suggestion
        - JM presented at EuroSciPy 2025, primarily [Python Profiling course](https://carpentries-incubator.github.io/pando-python/), however gave some visibility to sig-rpc in Europe
```

    - RSECon25 (Robert/Peter)
        - Large part of the SIG was to build the knowledge base / website
        - The workshop introduced the website and the SIG. 
        - Participants reviewed content, and opened isseus / Pull request for new / missing content.
          - ~40 PRs/Issues opened, reviewing / merging / implementing these in progress.
        - Lots of interesting converstions with people from differing backgrounds (who are newer to performacne etc)
            - Suggested to make the content more accessible to people less familiar / technical, i.e. intro on performance and how to use the website on the home page, call to action etc.
            - RC original structured the website to be broken up into selecting by language etc, but this was not natural for people who are newer to the area
            - Useful to have contacts with people from the range of backgrounds who can be asked to review changed versions.
        - Challenge to make the content accessible to the intended audience of users rather than contributors

>         - Lots of work to continue to build the website etc, difficult to find time as no allocation for this

Reviewer: ```suggestion
        - Lots of work to continue to build the website etc, difficult to find time as no allocation for this. Hopefully he will find time over Christmas to finish processing feedback from RSECon.
```

    - DIRAC HPC-AI Conference (Robert)

>         - RC presented on this in Leicster on 2025-10-15. More infrastructure oriented audience, some interesting conversations after the talk + new slack members. 

Reviewer: ```suggestion
        - RC presented on this in Leicester on 2025-10-15. More infrastructure oriented audience, some interesting conversations after the talk + new slack members. 
```

>         - One of the new network+ communities is interest in future collaboration with the SIG (potentially SHAREing)

Reviewer: ```suggestion
        - One of the new network+ communities may provide opportunities for future collaboration with the SIG (potentially [SHAREing](https://shareing-dri.github.io/))
        - [CAKE](https://www.cake.ac.uk/) have agreed to distribute some of our stickers at SuperCompute in the US in December
```

(The latter point I failed to mention!)

# 2025-10-16 AGM

2025 Annual General meeting

Agenda: [sig-rpc/minutes#7](https://github.com/sig-rpc/minutes/issues/7)

## Attendees

- (RC) Robert Chisholm [chair] (Sheffield)
- (PH) Peter Heywood [secretary] (Sheffield)

- (JM) Jost Migenda (KCL)
- (FS) Frederick Sonnenwald (Sheffield)
- (A) Alex (Oxford)
- (SC) Sam Cunliffe (UCL)
- (IB) Iain Barrass (Glasgow)
- (ST) Sam Tygier (STFC)

> - (AD) Abshishek Dasgupta (Oxford)

Reviewer 2: ```suggestion
- (AD) Abhishek Dasgupta (Oxford)
```
See https://www.rse.ox.ac.uk/dr-abhishek-dasgupta

            - Useful to have contacts with people from the range of backgrounds who can be asked to review changed versions.
        - Challenge to make the content accessible to the intended audience of users rather than contributors
        - Lots of work to continue to build the website etc, difficult to find time as no allocation for this. Hopefully he will find time over Christmas to finish processing feedback from RSECon.
    - DIRAC HPC-AI Conference (Robert)
        - RC presented on this in Leicester on 2025-10-15. More infrastructure oriented audience, some interesting conversations after the talk + new slack members. 
        - One of the new network+ communities may provide opportunities for future collaboration with the SIG (potentially [SHAREing](https://shareing-dri.github.io/))
        - [CAKE](https://www.cake.ac.uk/) have agreed to distribute some of our stickers at SuperCompute in the US in December
    - Undergraduate research project (Robert)
        - TUoS "Turing" module - 4th year group research project. 

>         - RC proposed a project for students to do a systematic / landscape review of performance in codes, Hoping to get a paper/dataset(s) from this to build larger future proposals

Reviewer 2: ```suggestion
    - JM to look it over and leave a comment/review on GitHub after the meeting
- Review of the year's activities
    - SSI CW/N8 RSE Meetup (Robert)
        - RC delivered some lightning talks at these events (short talks with a few slides) advertising the SIG
    - EuroSciPy (Jost)
        - JM presented at EuroSciPy 2025, primarily [Python Profiling course](https://carpentries-incubator.github.io/pando-python/), however gave some visibility to sig-rpc in Europe
    - RSECon25 (Robert/Peter)
        - Large part of the SIG was to build the knowledge base / website
        - The workshop introduced the website and the SIG. 
        - Participants reviewed content, and opened issues/PRs for new / missing content.
          - ~40 PRs/issues opened, reviewing / merging / implementing these in progress.
        - Lots of interesting conversations with people from differing backgrounds (who are newer to performance etc.)
            - Suggested to make the content more accessible to people less familiar / technical, i.e. intro on performance and how to use the website on the home page, call to action etc.
            - RC original structured the website to be broken up into selecting by language etc., but this was not natural for people who are newer to the area
            - Useful to have contacts with people from the range of backgrounds who can be asked to review changed versions.
        - Challenge to make the content accessible to the intended audience of users rather than contributors
        - Lots of work to continue to build the website etc., difficult to find time as no allocation for this. RC hopes he will find time over Christmas to finish processing feedback from RSECon.
    - DIRAC HPC-AI Conference (Robert)
        - RC presented on this in Leicester on 2025-10-15. More infrastructure oriented audience, some interesting conversations after the talk + new Slack members. 
        - One of the new network+ communities may provide opportunities for future collaboration with the SIG (potentially [SHAREing](https://shareing-dri.github.io/))
        - [CAKE](https://www.cake.ac.uk/) have agreed to distribute some of our stickers at SuperComputing in the US in December
    - Undergraduate research project (Robert)
        - TUoS "Turing" module - 4th year group research project. 
        - RC proposed a project for students to do a systematic / landscape review of performance in codes. Hoping to get a paper/dataset(s) from this to build larger future proposals
```
just a few typos

Reviewer: > SuperComputing 

I legit thought it was singular, it's too often just shortened to SC25

        - Students currently asked to select projects
- Other news:

>     - Jeremy Cohen at UCL from STEP-UP, RC not going to do this, but RC keen to support that, if anyone is keen get in touch via email / slack

Reviewer 2: ```suggestion
    - Jeremy Cohen at UCL sent invitation to a dRTP training landscape event by STEP-UP (London, 27th November). RC not going to do this, but RC keen to support that, if anyone is keen get in touch via email / Slack
```
add a bit more context

- Election of officers
    - > Minimum of 3 required, 2 of 4 current officers have resigned.
    - Chair: RC happy to continue as chair - no objections.
    - What do they do
        - PH has mostly been taking minutes. 
        - RC has been the driving force. Some of the larger SIGs have quartlery meetings wiht presentations.
        - Logner term associate with HPC-SIG / Green RSE sig to build collaborations. 
        - No official roles, just would be a steering / voting member for the SIG. 
        - Anything other than chair? 
            - Deputy Chair
            - Editor
            - Treasurer not needed at this stage but suggested.
        - JM volunteered to be an officer - could be deputy chair

>         - FW - happy to be a generic office ro keep the numbers

Reviewer 2: ```suggestion
- Election of officers
    - > Minimum of 3 required, 2 of 4 current officers have resigned.
    - Chair: RC happy to continue as chair - no objections.
    - Secretary: PH happy to continue as secretary - no objections.
    - What do officers do
        - PH has mostly been taking minutes. 
        - RC has been the driving force. Some of the larger SIGs have quarterly meetings with presentations.
        - Longer term associate with HPC-SIG / Green RSE sig to build collaborations. 
        - No official roles, just would be a steering / voting member for the SIG. 
        - Are there any specific roles, other than chair and secretary?
            - Deputy Chair
            - Editor
            - Treasurer (not needed at this stage but suggested)
    - JM volunteered to be an officer or deputy chair - no objections.
    - FW - happy to be a generic officer to keep the numbers - no objections.
```

Adding that PH continues as secretary. (I think that was quickly approved before the question/discussion part, but not 100% certain on the exact order.)

Other than that, just a bit of cleanup.

- Open discussion of plans for coming year
    - RC has 2 main goals
        - HPC-SIG meetings, probably quarterly. Some overlap but also differing opinions on level profiling 
    - Green RSE via Kirsty Pringle, some good conversations in the past
        - ST: Efficient computing / efficient software is part of Green, but it's complicated due to reboud effects etc. 
        - RC: 3 factors - productivity, mental heatlh, green (but this can just become )
            - ST how can you get carbon counting etc into jobs then that can drive optimisation
            - RC expect some back and forth with different groups to make content appropriate for them and present wider. 
    - AB: website stuff to increase the usability?
        - AB raised an issue during the RSE con about this
        - RC: having executing code in the browser as examples could be useful
        - RC: Someone else raised that the examples in the website were not fully explained / discussed, as its the general shape rather than exact values that were deemed important but this could be imrproved

>     - SC: Accessiubilty audit of the site? UCL have an office of people who could be contacted/asked

Reviewer 2: ```suggestion
        - ST: Efficient computing / efficient software is part of green computing, but it's complicated due to rebound effects etc.
        - RC: 3 factors - productivity, mental health, green (but this can just become )
            - ST how can you get carbon counting etc. into jobs then that can drive optimisation
            - RC expect some back and forth with different groups to make content appropriate for them and present more widely
    - AB: website stuff to increase the usability?
        - AB raised an issue during RSECon about this
        - RC: having executing code in the browser as examples could be useful
        - RC: Someone else raised that the examples in the website were not fully explained / discussed, as it’s the general shape rather than exact values that were deemed important but this could be improved
    - SC: Accessibility audit of the site? UCL have an office of people who could be contacted/asked
```
just a few typos
